src/components/BookComponent/AddBook.js

```diff
@@ -1,21 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+// import BookPage from '../../BookPage';
```

Comment:
- [OPTIONAL] Please kindly remove commented code from code base as this will make your code base much cleaner as crowded.